cmd/kube-proxy/app/server.go

```diff
@@ -263,12 +268,34 @@ func (o *Options) eventHandler(ent fsnotify.Event) {
 	eventOpIs := func(Op fsnotify.Op) bool {
 		return ent.Op&Op == Op
 	}
-	if eventOpIs(fsnotify.Write) || eventOpIs(fsnotify.Rename) {
+
+	if eventOpIs(fsnotify.Create) || eventOpIs(fsnotify.Write) || eventOpIs(fsnotify.Rename) {
```

Comment:
nit, now that we are monitoring the parent folder I don't know if we should filter the event by the file name to avoid processing unrelated events

`if ent.Name == oConfigFile && (eventOpIs(fsnotify.Create) || eventOpIs(fsnotify.Write) || eventOpIs(fsnotify.Rename))`

Reply:
Emm... This is hard to filter because the fs event name won't be `o.ConfigFile`, but some dynamic file names like `..2020_01_09_03_52_40.739411708`.  If we filter events by `if ent.Name == o.ConfigFile`, no action will be triggered.

Reply:
Forget it then, thought it was simpler